lighthouse-core/audits/byte-efficiency/legacy-javascript.js

```diff
@@ -436,6 +436,18 @@ class LegacyJavascript extends ByteEfficiencyAudit {
       };
       for (const match of matches) {
         const {name, line, column} = match;
+
+        let original;
```

Comment:
won't this be any?

```suggestion
        /** @type {LH.Details.SourceLocationValue['original']} */
        let original;
```

I think? might be another namesapce in there